### ISSUE TITLE
Unconventional sunrise and sunset times .

### ISSUE THREAD

**Issue by 00schteven** [basnijholt/adaptive-lighting#105]
I am trying to configure Adaptive lighting for my indoor garden. I would like lights to come on at 16:00 and off at 04:00. Setting manual Sunrise and sunset times in the configuration returns this error:

ValueError: Test: The sun events ('sunset', 'solar_noon', 'sunrise', 'solar_midnight') are not in the expected order. The Adaptive Lighting integration will not work! This might happen if your sunrise/sunset offset is too large or your manually set sunrise/sunset time is past/before noon/midnight.

I think I can set the server location or clock to a time zone more compatible, but it would be nice to maintain a local time zone for the rest of the system.

**Comment by RubenKelevra** [comment]
I agree, this should be possible and the verification code for the time input is probably too restrictive

**Comment by basnijholt** [comment]
I never considered this use case. It should indeed be possible to support this case.

**Comment by th3w1zard1** [comment]
@00schteven I believe I've implemented this in #535 

if someone could test, that'd be great!

**Comment by b-rad15** [comment]
Been a while since this lol but I'm testing using this version now and it appears to be working

**Comment by b-rad15** [comment]
Rebased this in #830

**Comment by basnijholt** [comment]
The 16:00 to 04:00 case works with current code. #1535 adds a tested configuration and power-schedule example, including startup handling. Closing as resolved.
